qube.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.12.0-SNAPSHOT
+current_version = 1.12.0
```

Comment:
did you use qube to bump the version?

Reply:
Yes i used the qube `bump-version` command with the qube 2.5.1 version to increase the versioning of `data-model-lib`
Interesting that it skipped to do so in the `qube.cfg `Nice Catch 👍

Reply:
No its fine there is no problem. I was just curious in general.